backend/market/market.go

```diff
@@ -181,11 +182,11 @@ func ListExchangesByRegion(account accounts.Interface, httpClient *http.Client)
 		})
 	}
 
-	return exchangeRegions
+	return vendorRegions
 }
 
-// GetExchangeDeals returns the exchange deals available for the specified account, region and action.
-func GetExchangeDeals(account accounts.Interface, regionCode string, action ExchangeAction, httpClient *http.Client) ([]*ExchangeDealsList, error) {
+// GetMarketDeals returns the vendor deals available for the specified account, region and action.
+func GetMarketDeals(account accounts.Interface, regionCode string, action Action, httpClient *http.Client) ([]*DealsList, error) {
```

Comment:
I think that to avoid stutter, we should call this `GetDeals`, imho `market.GetDeals` is better than the repeating `market.GetMarketDeals`